EdFi.Ods.Utilities.Migration/logistics/scripts/modules/build-edfi-standard-descriptors.psm1

```diff
@@ -0,0 +1,105 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+$ErrorActionPreference = "Stop"
+Import-Module -Force -Scope Global (Resolve-Path "$PSScriptRoot/tasks/TaskHelper.psm1")
+
+function New-EdFiStandardDescriptorsPackage {
+    param(
+        [string]
+        $OutputPath = (Join-Path $env:temp 'EdFi.Standard.Descriptors/'),
+
+        # NOTE: the target folder version is the api version the migration utility is migrating to NOT the data standard version
+        [hashtable[]]
+        $PackageFiles = @(
+            @{ nuspecsrc = (Resolve-Path "$PSScriptRoot/../../../EdFi.Standard.Descriptors/");src = (Resolve-Path "$PSScriptRoot/../../../../../Ed-Fi-Standard/v3.1/Descriptors/"); target = (Join-Path $OutputPath "any/netcoreapp3.1/Descriptors/3.1") }
+        )
+    )
+
+    Write-Host "OutputPath $OutputPath"
+    $PackageFiles | % { Write-HashtableInfo $_ }
+
+    $config = @{
+        OutputPath = $OutputPath
+        PackageFiles = $PackageFiles
+    }
+
+    $script:tasks = @(
+        'Remove-TempDirectory'
+        'New-TempDirectory'
+        'Copy-PackageFiles'
+        'Copy-PackageNuspec'
+    )
+
+    $script:result = @()
+
+    $elapsed = Use-StopWatch {
+        foreach ($task in $tasks) {
+            $script:result += Invoke-Task -name $task -task { & $task $config }
+        }
+    }
+
+    Test-Error
+
+    $script:result += New-TaskResult -name '-' -duration '-'
+    $script:result += New-TaskResult -name $MyInvocation.MyCommand.Name -duration $elapsed.format
+
+    return $script:result | Format-Table
```

Comment:
> These functions are required for build-edfi-standard-descriptors.psm1

what's preventing them from being removed? would something like this work?
```suggestion
    Remove-TempDirectory $config
    New-TempDirectory $config
    Copy-PackageFiles $config
    Copy-PackageNuspec $config
```

Reply:
Yes I changed it . Thanks